src/macro-expansion.md

```diff
@@ -132,8 +132,8 @@ algorithm is exactly the same, but when the macro parser comes to a place in the
 current matcher where it needs to match a _non-terminal_ (e.g. `$mvar:ident`),
 it calls back to the normal Rust parser to get the contents of that
 non-terminal. In this case, the Rust parser would look for an `ident` token,
-which it finds (`foo`) and returns to the macro parser. Then, the macro parser
-proceeds in parsing as normal. Also, note that exactly one of the matchers from
+which it finds (`foo`), and returns to the macro parser. Then, the macro parser
```

Comment:
"returns to the macro parser" is doesn't have a subject and a verb, so I think there should be no comma.

Reply:
Yeah, I think you're right actually. The `foo` in brackets just broke up the flow a bit. Will revert.